force-app/main/default/classes/SelectOptionSorter.cls

```diff
@@ -0,0 +1,66 @@
+public with sharing class SelectOptionSorter 
+{
+    public static void sortSelectOptionList( List<SelectOption> selectOptionList )
+    {
+        Contract.requires( selectOptionList != null, 'sortSelectOptionList called with null selectOptionList' );
+
+        Map<String, String> optionListMap = new Map<String, String>(); 
+
+        for( SelectOption so : selectOptionList )  
+        {
+            String labelKey = so.getLabel(),
+
+                      value = so.getValue();
```

Comment:
why cant I see String with value?

Reply:
There's a comma. Both labelKey and value have the same types so I wrote String once